tests/core/test_opds2_import.py

```diff
@@ -407,3 +409,18 @@ def test_opds2_importer_skips_publications_with_unsupported_identifier_types(
         # Ensure that it was parsed correctly and available by its identifier.
         edition = self._get_edition_by_identifier(imported_editions, identifier)
         assert edition is not None
+
+    def test_auth_token_feed(self):
+        content = self.sample_opds("pq_auth_token_feed.json")
```

Comment:
Since we have this feed as a fixture. It would be nice to have a test that tests one of the items in the feed all the way to fulfillment. We have unit tests for each piece, but since we're importing the feed, we should be able to do a more holistic test as well.